manifests/z/zokugun/MrCode/1.82.0.23253/zokugun.MrCode.locale.en-US.yaml

```diff
@@ -4,7 +4,7 @@
 PackageIdentifier: zokugun.MrCode
 PackageVersion: 1.82.0.23253
 PackageLocale: en-US
-Publisher: Microsoft Corporation
+Publisher: zokugun

```

Comment:
```suggestion
Publisher: Zokugun
```
It would be best if it matched the same capitalization as Control Panel.

![image](https://github.com/microsoft/winget-pkgs/assets/11600822/a00a5c2e-ba3b-4273-b5fb-2de0b5a32653)